transport/mapper.go

```diff
@@ -3,20 +3,20 @@ package transport
 import (
 	"context"
 
-	"github.com/ThreeDotsLabs/esja/stream"
+	"github.com/ThreeDotsLabs/esja"
 )
 
 // Mapper translates the event into a serializable transport model.
 type Mapper[T any] interface {
-	// New returns an instance of a transport model
-	// corresponding to provided stream.EventName.
-	New(stream.EventName) (any, error)
+	// New returns a new instance of a transport model
+	// corresponding to the provided event name.
+	New(string) (any, error)
```

Comment:
```suggestion
	New(eventName string) (any, error)
```

Reply:
Since it is a string, it makes sense. The type does not explain the meaning anymore. I will fix that.

Reply:
Fixed